src/taraxad/taraxad_version.hpp

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+// get version of cmake project
+#define TARAXAD_VERSION "1.9.0"
```

Comment:
I believe we dont use this file anymore...

Reply:
yes, this file should not be committed